### ISSUE TITLE
Problem with T2 values

### ISSUE THREAD

**Issue by Empanag** [UMEP-dev/UMEP#266]
**Describe the bug**
hi, i'm using SUEWS for a grid in city of Heraklion at Crete. The model produce some values from the T 2m that are very low, reaching -40 C that is for sure unreal.

**Desktop (please complete the following information):**
 - OS: windows 10
 - SuPy Version 2021.2.3 (I m using python compiler not QGIS)

**Screenshots**
![image](https://user-images.githubusercontent.com/61833032/112841250-f092f380-90a8-11eb-8b2f-f7a7cc967fc0.png)

**Data**
[suews.zip](https://github.com/UMEP-dev/UMEP/files/6221853/suews.zip)

**Comment by sunt05** [comment]
@Empanag Please update SuPy to a recent version and see if this issue persists.

**Comment by Empanag** [comment]
yes again some values are at -20,-19. I run it with the newest 2021.3.24 but in different pixels of the grid.
